xikolo-ios/ViewControllers/Courses/CourseContentTableViewController.swift

```diff
@@ -55,13 +63,23 @@ class CourseContentTableViewController: UITableViewController {
         CourseSectionHelper.syncCourseSections(course).flatMap { sections in
             sections.map { section in
                 CourseItemHelper.syncCourseItems(section)
-            }.sequence().onComplete { _ in
-                self.tableView.reloadEmptyDataSet()
-                self.preloadCourseContent()
+                }.sequence().onComplete { _ in
+                    self.tableView.reloadEmptyDataSet()
+                    self.preloadCourseContent()
             }
-        }.onComplete { _ in
-            NetworkIndicator.end()
+            }.onComplete { _ in
+                NetworkIndicator.end()
         }
+
+    }
+    
+    override func viewDidLoad() {
```

Comment:
Can move the method up a little bit (before `loadData` and `handleRefresh`)? Because it's a very common method.